src/ui/public/inspector/ui/inspector_view_chooser.js

```diff
@@ -72,7 +77,7 @@ class InspectorViewChooser extends Component {
         onClick={this.toggleSelector}
         data-test-subj="inspectorViewChooser"
       >
-        View: { this.props.selectedView.title }
+        {viewLabel}: { this.props.selectedView.title }
```

Comment:
Since we cannot be sure about the "ordering" or words and if the colon is the separator in all languages, this should be one string (using `<FormattedMessage>`) that gets `this.props.selectedView.title` as a placeholder value, so that the i18n string will be `View: {viewTitle}` in the end. Same below.